dbt-cta/dbt_project.yml

```diff
@@ -62,11 +62,26 @@ models:
 
     # for the /monitoring project
     0_staging:
+      +tags:
```

Comment:
Are these being used by other models? Like this wont accidentally grant access to some tables we dont want granted access to?

Reply:
sweet thanks for the helpful commentary!